sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTransmitter.cs

```diff
@@ -79,10 +79,10 @@ internal async ValueTask<int> AddBatchActivityAsync(Batch<Activity> batchActivit
             return response.Value.ItemsAccepted.GetValueOrDefault();
         }
 
-        private static TelemetryEnvelope GeneratePartAEnvelope(Activity activity)
+        private static TelemetryItem GeneratePartAEnvelope(Activity activity)
         {
             // TODO: Get TelemetryEnvelope name changed in swagger
-            TelemetryEnvelope envelope = new TelemetryEnvelope(PartA_Name_Mapping[activity.GetTelemetryType()], activity.StartTimeUtc);
```

Comment:
I think you can remove this TODO.
Also, this variable should be renamed to match the type.